tracer/build/_build/Build.Gitlab.cs

```diff
@@ -23,7 +23,7 @@ partial class Build
        .Description("Sign the dlls produced by building the Tracer, Profiler, and Monitoring home directory, as well as the dd-dotnet exes")
        .Unlisted()
        .Requires(() => IsWin)
-       .After(BuildTracerHome, BuildProfilerHome, BuildNativeLoader, CreateRootDescriptorsFile, BuildDdDotnet, CopyDdDotnet)
+       .After(BuildTracerHome, BuildProfilerHome, BuildNativeLoader, CreateMissingNullabilityFile, CreateRootDescriptorsFile, BuildDdDotnet, CopyDdDotnet)
```

Comment:
I don't think we need to do this in gitlab 🤔 We only do the root descriptors here because we pack it into a NuGet

```suggestion
       .After(BuildTracerHome, BuildProfilerHome, BuildNativeLoader,  CreateRootDescriptorsFile, BuildDdDotnet, CopyDdDotnet)
```